aws/templates/id/id_mobilenotifier.ftl

```diff
@@ -0,0 +1,116 @@
+[#-- MOBILENOTIFIER --]
+
+[#-- Components --]
+[#assign MOBILENOTIFIER_COMPONENT_TYPE = "mobilenotifier" ]
+[#assign MOBILENOTIFIER_PLATFORM_COMPONENT_TYPE = "mobilenotiferplatform" ]
+
+[#assign componentConfiguration +=
+    {
+        MOBILENOTIFIER_COMPONENT_TYPE : {
+            "Attributes" : [
+                { 
+                    "Name" : "Links",
+                    "Subobjects" : true,
+                    "Children" : linkChildrenConfiguration
+                },
+                {
+                    "Name" : "SuccessSampleRate",
+                    "Default" : "100"
+                },
+                {
+                    "Name" : "Credentials",
+                    "Children" : [
+                        {
+                            "Name" : "EncryptionScheme",
+                            "Default" : "base64"
+                        }
+                    ]
+                }
+            ],
+            "Components" : [
+                {
+                    "Type" : MOBILENOTIFIER_PLATFORM_COMPONENT_TYPE,
+                    "Component" : "Platforms",
+                    "Link" : [ "Platform" ]
+                }
+            ]
+        },
+        MOBILENOTIFIER_PLATFORM_COMPONENT_TYPE : [
+            {
+                "Name" : "Engine"
+            },
+            {
+                "Name" : "SuccessSampleRate"
```

Comment:
what's the logic for these being on the notifier and each platform? Presume it aligns with AWS ...

Reply:
Allows for a global mobilenotifier setting and overrides on the platforms. So if you have multiple platforms you can configure the sample rate individually

Reply:
Normal override behaviour for settings should handle that though. So set on the component and it will be available for all subcomponents.